lang/en/docs/tutorials/materials/specific/defect-point-vacancy-boron-nitride.md

```diff
@@ -0,0 +1,138 @@
+---
+# YAML header
+render_macros: true
+---
+
+# Vacancy Point Defects in Hexagonal Boron Nitride.
+
+## Introduction.
+
+This tutorial demonstrates the process of creating materials with vacancy point defects, based on the work presented in the following manuscript:
+
+!!!note "Manuscript"
+    Fabian Bertoldo, Sajid Ali, Simone Manti & Kristian S. Thygesen, "Quantum point defects in 2D materials - the QPOD database", Nature, 2022. [DOI:10.1038/s41524-022-00730-w](https://doi.org/10.1038/s41524-022-00730-w){:target='_blank'}.
+
+We use the [Materials Designer](../../../materials-designer/overview.md) and JupyterLite environment to create a nanoribbon of hexagonal boron nitride (hBN) and introduce vacancy defects. The process combines the capabilities of nanoribbon creation and point defect introduction.
+
+We will focus on creating a structure similar to Figure 6 from the manuscript, which demonstrates boron vacancy defects in hexagonal boron nitride:
+
+![Vacancy in hBN](/images/tutorials/materials/defects/defect_point_vacancy_boron_nitride/0-figure-from-manuscript.webp "Vacancy in hBN")
+
+## 1. Import Base Material.
+
+First, we need to import the hexagonal boron nitride (hBN) material from the [Standata](../../../materials-designer/header-menu/input-output/standata-import.md) database.
+
+### 1.1. Open Materials Designer.
+
+Navigate to [Materials Designer](../../../materials-designer/overview.md) and click on the "Input/Output" menu.
+
+### 1.2. Import from Standata.
+
+1. Select "Import from Standata" in the Input/Output menu
+2. In the search box, enter "Boron Nitride"
+4. Click on the hBN material to import it
+
+![Standata hBN Import](/images/tutorials/materials/interfaces/twisted-bilayer-boron-nitride/standata-import-bn.png "Standata hBN Import")
+
+## 2. Create hBN Nanoribbon.
+
+Next, we'll create a nanoribbon structure using the JupyterLite environment.
+
+### 2.1. Launch JupyterLite Session.
+
+Select the "Advanced > [JupyterLite Transformation](../../../materials-designer/header-menu/advanced/jupyterlite-dialog.md)" menu item to launch the JupyterLite environment.
+
+![JupyterLite Dialog](/images/jupyterlite/md-advanced-jl.webp "JupyterLite Dialog")
+
+### 2.2. Open and Configure Nanoribbon Notebook.
+
+Find and open `create_nanoribbon.ipynb` in the list of notebooks. Edit the nanoribbon parameters in section 1.1 of the notebook:
+
+```python
+WIDTH = 8  # in number of atoms
+LENGTH = 14  # in number of atoms
+VACUUM_WIDTH = 0  # in number of atoms
+VACUUM_LENGTH = 0 # in number of atoms
```

Comment:
How are these distances measured in the number of atoms? Not clear - is this number of layers?

Reply:
Yes, layers, but perpendicular to Z-axis

Reply:
atoms -> unit cells